src/ports/rt-kernel/pnal.c

```diff
@@ -169,26 +163,26 @@ static pnal_eth_mau_t calculate_mau_type (uint8_t link_state)
 /**
  * Calculate advertised capabilites
  *
- * @param anar       In:    Contents of the ANAR register read from PHY.
+ * @param rtkernel_capabilities In:    Contents of the ANAR register read from PHY.
```

Comment:
advertised or advertised_capabilities? The capabilities are not related to rt-kernel?

Reply:
Changed to `capabilities` as to match defines. Also updated parameter comment as to clarify.